local-interchain/README.md

```diff
@@ -0,0 +1,135 @@
+# Local Interchain
+
+A simple way to config and run IBC local chain testing environments with any language of choice for interaction.
+
+## Installing
+
+**Install on Mac / Linux**
+```bash
+git clone https://github.com/strangelove-ventures/interchaintest.git 
+
+cd local-interchain
+
+# NOTE: your binary will link back to this location of where you install.
+# If you rename the folder or move it, you need to `make install` the binary again.
+make install
+```
+**Install on Windows**
+
+Follow [this guide](./docs/WINDOWS.md) to setup the Windows OS environment for installing Local Interchain.
+
+## Running
+
+- *(optional)* Edit `./configs/relayer.json`
+- Copy: `cp ./configs/base_ibc.json ./configs/mytest1_ignored.json`
+- Run: `local-ic start mytest1_ignored.json`
+- Change directory `ICTEST_HOME=/root/example/local-interchain local-ic start myother_ignored.json`
```

Comment:
I think a little bit of clarity between these two bullets here could help users get started.

Changing directory appears to be optional. If user does change directory, they have to create a `chains` folder in there right? To house the config?
(I commented about this somewhere else as well)


Potentially open to the idea of simplifying and having it just accept full paths.
Ex: `local-ic start full/path/to/config.json`

Reply:
Yes, if you move your configs elsewhere you would need to have a `chains` folder in the ICTEST_HOME directory. Good point

I was trying to get away with not using full/abs paths since it is meant to be quick `local-ic start NAME` vs needing to always /home/name/.../chain.json

maybe default is to search in the chains/ folder, if not there use absolute / full path or something

Reply:
Resolved in: eff98a168bb87af75d0e326c4a4aaab20d96acf7

Requires chains and config folder in the selected path or else it panics. Also added to docs to explain